libevm/VM.h

```diff
@@ -101,13 +103,18 @@ class VM: public VMFace
 	u256s m_stackSpace;
 	u256* m_stack;
 	
+	// space for return stack and pointer to data
+	std::vector<uint64_t> m_returnSpace;
+	uint64_t* m_return;
```

Comment:
I still think it is better to put a `=nullptr` here in case someone creates a new constructor and forgets to initialize that member.